### PR TITLE
[Edit] C++: Loops

### PR DESCRIPTION
**Pull request by bordenmike518**
### Description

I added some more information about how the syntax of a for loop can be altered to achieve different results. One example is to leave the for loop empty which will create a for loop. Another interesting point to make is that all the parameters of a for loop do not need to be a made inside the parameter space.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/loops/loops.md [hunk 1]
```diff
@@ -44,4 +44,21 @@ for (int i = 0; i < 10; i++)
...
+```
+
+An empty `for` loop would result in an infinite loop. For example:
+
+```
+for ( ; ; )
+{
+  // Infinite loop
+}
 ```
```
Comment (Dusch4593) [inline content/cpp/concepts/loops/loops.md:64]:
```suggestion
```

I wouldn't recommend adding code for an infinite loop.

File: content/cpp/concepts/loops/loops.md [hunk 2]
```diff
@@ -44,4 +44,21 @@ for (int i = 0; i < 10; i++)
 {
   std::cout << i << "\n";
 }
+
+// This would also run the same as above.
+int i = 0;
+for( ; i < 10; )
+{
```
Comment (Dusch4593) [inline content/cpp/concepts/loops/loops.md:51]:
```suggestion
for( ; i < 10; ) {
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hey @bordenmike518! Thank you for contributing to Codecademy Docs! 😄

I had some feedback to offer, and some more to follow. But first, I wanted to just ask if you happened to [push](https://github.com/Codecademy/docs/pull/1260/commits/d449f55c1750ee6a274e156efea3ed10c2f3bc84) from a possibly different account named "Borden"?

**Comment by Dusch4593** [comment]
Hey @bordenmike518 it's been a couple of weeks of inactivity on this PR. Did you have any questions or need any assistance with [the feedback](https://github.com/Codecademy/docs/pull/1260#pullrequestreview-1185379234) I gave?

**Comment by yangc95** [comment]
@bordenmike518 I'm not actually sure about the lines of code that were added here. It doesn't look like good practice even though it's possible to omit certain parameters,

What do we think? @Dusch4593 @HishamT

**Comment by yangc95** [comment]
Closing out this PR as the change does not seem to be good practice.

**Comment by yangc95** [comment]
@bordenmike518 apologies, upon further evaluation, it looks like it's a common practice among developers to omit certain parameters like this (it applies for other languages as well like Swift).

File: content/cpp/concepts/loops/loops.md [hunk 3]
```diff
@@ -42,4 +42,11 @@ This example prints 0 to 9 on the screen:
 for (int i = 0; i < 10; i++) {
   std::cout << i << "\n";
 }
+
+// This would also run the same as above.
+int i = 0;
+for( ; i < 10; ) {
+  std::cout << i << "\n";
+  i++;
```
Comment (yangc95) [inline content/cpp/concepts/loops/loops.md:50]: @bordenmike518 I would also like to see something about how we can add more than one condition and action.

`i++;` can go here or in the parenthesis.

File: content/cpp/concepts/loops/loops.md [hunk 4]
```diff
@@ -42,4 +42,11 @@ This example prints 0 to 9 on the screen:
 for (int i = 0; i < 10; i++) {
   std::cout << i << "\n";
 }
+
+// This would also run the same as above.
+int i = 0;
+for( ; i < 10; ) {
```
Comment (yangc95) [inline content/cpp/concepts/loops/loops.md:48]:
```suggestion
for ( ; i < 10; ) {
```

**Comment by Dusch4593** [comment]
Since there's been a few weeks of inactivity on this PR, and because the CLA still has yet to be signed, I am closing it for now. Let us know if you have any questions about how we can reconcile this issue with the two GitHub usernames and we'll be happy to help. 😄

**Pull request closed without merging**